API/Login.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+	ini_set('display_errors', 1);
+    ini_set('error_reporting', E_ALL);
+
+	$inData = getRequestInfo();
+
+	$servername = "localhost";
+	$username = "root";
+	$password = "";
+	$dbname = "myDB";
+
+	// Create connection
+	$conn = new mysqli($servername, $username, $password, $dbname); 	
+	if( $conn->connect_error )
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		$userExists = userExists($conn, $inData['username']);
+
+		if ($userExists === false){
+			flush();
+			returnWithError("User doesn't exist.");
+			return;
+		}
+
+		$checkPass = password_verify($inData['password'], $userExists['password']);
```

Comment:
Instead of using password_verify, just compare the sent password to the stored password (because we store the hashed password and send a hashed password).